src/database.js

```diff
@@ -128,6 +128,32 @@ function setUpModels(db) {
             }
         }
     });
+    models.Pod.homeStats = function (callback) {
+        db.driver.execQuery(
+          "SELECT \
+            (SELECT COUNT(*) FROM pods) AS total_nodes,\
```

Comment:
I fixed the current failure test but I had in mind that these queries are a temporary hack, as the use of CURDATE() doesn't even return anything between midnight and the moment the data are inserted :p So definitely broken but I'll have a look once the front-end is fine.